apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1255,6 +1256,38 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.getNames',
```

Comment:
I'm not totally against this, but I was wondering if we shouldn't be able to use `/api/v1/users.list` for this purpose.. 

or maybe, even better, we could have an endpoint within the "messages" namespace to get the reactions list from it.. this way we don't need to provide a list of users to get some info, we'd provide just the message id.. wdyt?

Reply:
`users.list` is a paginated API, also doesn't return for a set of particular users like we need in this case.

Reply:
I like the idea of having such implementation under `messages` namespace and using the `messageId` to get normalized reactions list, but it would have been more expensive to fetch the message reactions using messageId and then querying for user's details, what do you think about this, as this API will be called a lot of times?

Reply:
Yep, also looks like a good idea to me to add a new `messages` endpoint for this at some point
Since we must consider heavy usage for this now, it might be better to keep this new endpoint as it is and avoid this extra db trip :thinking:

Reply:
having this endpoint being called multiple times with a single (and constant) parameter is a ideal scenario for caching.. in the other hand, if we'd like to add cache to `users.getNames` endpoint it would not be as efficient since it would receive a lot of different combination of params..

btw, the payload query for `users.getNames` doesn't scale well.. it is kinda rare having too many reactions in a single message, but there are no limits still.. so you may need to provide a list of hundreds of user IDs as the payload.. not ideal too..

other reasons I'm also not a fan of the `users.getNames` is that we're creating another very specific endpoint.. that's the point when I mentioned the `users.list` endpoint, it _should_ be able to provide this data, but unfortunately it doesn't =/ in this same principle, maybe we could just improve/expand the existing `chat.getMessage` endpoint to return this data, maybe..

Reply:
As suggested, now using `chat.getMessage` API to get normalized reactions.